test/unit/transaction-runner-test.coffee

```diff
@@ -1092,6 +1137,23 @@ describe 'TransactionRunner', ()->
           assert.ok configuration.emitter.emit.calledWith "test fail"
           done()
 
+      it 'should add fail message as a error under `general` to the results on transaction', (done) ->
+        runner.executeAllTransactions [transaction], runner.hooks, () ->
+          messages = transaction['results']['general'].map (value, index) -> value['message']
+          assert.include messages.join(), 'expected false to be truthy'
+          done()
+
+      it 'should add fail message as a error under `general` to the results on test passed to the emitter', (done) ->
```

Comment:
dtto - isn't checking `severity` of the message although the `it` message says so